modular-docs-manual/content/topics/assembly_creating_procedure_modules.adoc

```diff
@@ -0,0 +1,13 @@
+[[creating-procedure-modules]]
+= Creating Procedure Modules
+
+This section explains what a procedure module is and provides recommended practices for writing procedure modules.
+
+include::module_definition-procedure.adoc[leveloffset=+1]
+
+include::module_guidelines-procedure.adoc[leveloffset=+1]
+
+== Related Information
```

Comment:
I thought the consensus was to call this section "Additional Resources". I believe it's more to the point.